services/slam/server.go

```diff
@@ -143,6 +144,23 @@ func (server *subtypeServer) GetInternalState(ctx context.Context, req *pb.GetIn
 	}, nil
 }
 
+// GetPointCloudMapStream calls the callback function returned from the request to the slam service's
```

Comment:
Given that this is a GRPC server method I don't we should only document what this does from the caller's perspective, not how it does it. 

Recommended change:
// GetInternalStateStream returns the internal state of the slam service's slam algo in a stream of byte chunks.